docs/ConfigurationFile.md

```diff
@@ -119,6 +119,23 @@ exports.config = {
     // Level of logging verbosity: trace | debug | info | warn | error
     logLevel: 'info',
     //
+    // Set specific log levels per logger
+    // use 'silent' level to disable logger
```

Comment:
I think we need more documentation on which logger are available. I am not sure if the logger name is always the package name. This can confuse people.

Reply:
There are following loggers

- webdriver
- webdriverio

services:
- wdio-applitools-service
- wdio-browserstack-service
- wdio-devtools-service
- wdio-sauce-service

test frameworks
- wdio-mocha-framework
- wdio-jasmine-framework

runners
- wdio-local-runner
- wdio-lambda-runner

reporters
- wdio-sumologic-reporter

@wdio utils
- wdio-cli
- wdio-config
- wdio-sync
- wdio-utils

---

do you think we should keep it somewhere in docs? Users may configure particular logger simply depending on output.

Reply:
some packages, like allure-reporter, reusing existing logger instances, ex wdio-runner.

Reply:
@christian-bromann updated everything except of this

Reply:
not sure if it is a proper place for loggers list
https://github.com/webdriverio/webdriverio/pull/3467/files#diff-bad5e7d995f0493ee7a0c93b3090b0b3R107

Reply:
What about loggers like this https://github.com/webdriverio/webdriverio/blob/00bdd480d34757277d8b9f4f62f504311671a82e/packages/wdio-devtools-service/src/commands.js#L11 , I am sure we will have more specific loggers, how do we manage these?

Reply:
if you do something like this
logLevels {
  'wdio-devtools-service': 'silent'
}
it will change log levels for 
wdio-devtools-service
and every logger that starts with wdio-devtools-service:
ex: wdio-devtools-service:CommandHandler

is it what are you asking for?

Reply:
https://github.com/webdriverio/webdriverio/pull/3467/files#diff-df13ae3e41b9e0b3e6427a57619a38c4R119